test_patching_tool_version.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+source _liferay_common.sh
```

Comment:
@lucasmiranda0, this test isn't being collected by the `run_tests.sh`, can you please fix it?

A possible fix is:
```
function main {
	local tests_results=$(\
		find . -name "test_build_*.sh" -or -name "test_patching_tool_version.sh" -type f -exec ./{} \; && \
		\
		cd release && \
		\
		find . -name "test_*.sh" -type f -exec ./{} \;)

	echo "${tests_results}"

	if [[ "${tests_results}" == *"FAILED"* ]]
	then
		exit 1
	fi
}
```